company_linkage/parat_scripts/get_ai_counts.py

```diff
@@ -160,50 +162,57 @@ def run_query_id_papers(self, table_name: str, test: bool = False) -> list:
         :param test: False if not running as a unit test
         :return:
         """
-        companies_query = f"""SELECT CSET_id, ror_id FROM 
-        `gcp-cset-projects.high_resolution_entities.aggregated_organizations`"""
-        if test:
-            companies_query += """ LIMIT 25"""
+        companies_query = f"""-- SELECT CSET_id, ror_id FROM 
```

Comment:
Is there a reason to leave this stuff commented vs just deleting it?

Reply:
Nope; deleting.